.github/CODEOWNERS

```
@@ -17,7 +19,7 @@
 /cmd/process-agent/                     @DataDog/burrito
 /cmd/system-probe/                      @DataDog/burrito
 
-/docs/                                  @DataDog/baklava @DataDog/agent-core
+/docs/                                  @DataDog/baklava @DataDog/agent-platform
 
```

I'd give agent-core ownership of the `/docs/dev/checks/` directory, as it contains rtloader-related docs. 
```suggestion
/docs/dev/checks/                       @DataDog/baklava @DataDog/agent-core

```